src/workloads/execution/BackgroundTTLDeletions.yml

```diff
@@ -0,0 +1,187 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/server-execution"
+Description: |
+  This workload tests the impact of background TTL deletions in a heavily modified collection
+
```

Comment:
Brainstorming here --- Do we think it would be valuable to test the write throughput when there are writes on a secondary, non-ttl collection as well? 

The TTLMontior will be acquiring locks for the specific TTL collection. Since ticket contention is handled at the GlobalLock level, maybe we want to test write throughput on a non-TTL collection as well? 

I'm not sure if we would want to do that in this test, or another test completely/ if it is possible to test this reliably given the TTLMonitors sleeping nature.

Reply:
I'm not sure it would matter that much. Both TTL and inserts acquire a MODE_IX lock on the collection so they will functionally do the same regardless of having a TTL or not. The only thing that would serve is to observe the overhead of modifying the same collection too much in WiredTiger.